server/portal/apps/workspace/api/views.py

```diff
@@ -113,14 +112,13 @@ def get(self, request, *args, **kwargs):
         else:
             limit = int(request.GET.get('limit', 10))
             offset = int(request.GET.get('offset', 0))
-            # TODOv3: Query portal
-            # portal_name = settings.PORTAL_NAMESPACE
+            portal_name = settings.PORTAL_NAMESPACE
 
             data = tapis.jobs.getJobSearchList(
                 limit=limit,
                 startAfter=offset,
-                orderBy='lastUpdated(desc),name(asc)'
-                # _tapis_query_parameters={'tags.contains': portal_name}
+                orderBy='lastUpdated(desc),name(asc)',
+                _tapis_query_parameters={'tags.contains': portal_name}
```

Comment:
Is there a plan to add searching on tags or other fields by a keyword sent through the request? Or would that be part of a different task, perhaps [FP-1832](https://jira.tacc.utexas.edu/browse/FP-1832)?

Reply:
Hmm 🤔 
This might be unrelated, but I was wondering how job listing is different from the [Job History](https://tapis-project.github.io/live-docs/?service=Jobs#tag/jobs/operation/getJobHistory) (the v3 endpoint vs our job history view).

But, I think the task/feature you are pointing out is a different.
Do you think explicit searching via keyword should be on the same endpoint?

Reply:
I think it could be a different task. But, I'll let Shayan know!

Reply:
@shayanaijaz I think we would implement that as a part of FP-1832 - though certainly we might want to consider using the same django endpoint with added request params if it makes sense!